tutorials/notebooks/example_keras_mobilenet.py

```diff
@@ -104,3 +105,15 @@ def representative_data_gen() -> list:
     quantized_model, quantization_info = mct.ptq.keras_post_training_quantization_experimental(model,
                                                                                            representative_data_gen,
                                                                                            target_platform_capabilities=target_platform_cap)
+
+    # Export quantized model to TFLite
+    _, tflite_file_path = tempfile.mkstemp('.tflite') # Path of exported model
+    mct.exporter.keras_export_model(model=quantized_model, save_model_path=tflite_file_path,
+                                    target_platform_capabilities=target_platform_cap,
+                                    serialization_format=mct.exporter.KerasExportSerializationFormat.TFLITE)
+
```

Comment:
It's better to export tflite in int8 dtype. So, please add to the comment above tflite export something like: "TFLite can be exported in int8, please see more here" regarding the exporter README.
I think this comment should be in all Keras tutorials.